code/game/objects/structures/statues.dm

```diff
@@ -8,12 +13,14 @@
 	max_integrity = 100
 	var/oreAmount = 5
 	var/material_drop_type = /obj/item/stack/sheet/metal
+	var/impressiveness = 15
 	CanAtmosPass = ATMOS_PASS_DENSITY
 
 
 /obj/structure/statue/attackby(obj/item/W, mob/living/user, params)
 	add_fingerprint(user)
-	user.changeNext_move(CLICK_CD_MELEE)
+	if(!do_after(user, 20, target = src))
```

Comment:
You can remove this check safely, there's no reason for this one. You don't admire in this path.